raiden/tests/integration/contracts/test_payment_channel.py

```diff
@@ -0,0 +1,157 @@
+import gevent
+from eth_utils import (
+    to_canonical_address,
+    encode_hex,
+    decode_hex,
+    to_checksum_address,
+)
+from raiden_libs.utils.signing import sign_data
+from raiden_libs.messages import BalanceProof
+
+
+from raiden.network.rpc.client import JSONRPCClient
+from raiden.network.proxies import TokenNetwork, PaymentChannel
+from raiden.constants import (
+    NETTINGCHANNEL_SETTLE_TIMEOUT_MIN,
+    EMPTY_HASH,
+)
+
+
+def wait_blocks(web3, blocks):
```

Comment:
this is duplicate of the same function from `test_token_network`, right? Then it should go to i.e. `raiden.tests.utils` to minimize duplicated code.